amundsen_application/static/js/components/DashboardPage/index.tsx

```diff
@@ -184,8 +219,12 @@ export class DashboardPage extends React.Component<DashboardPageProps, Dashboard
                 <div className="section-title title-3">Last Run</div>
                 <div>
                   { formatDateTimeShort({ epochTimestamp: dashboard.last_run_timestamp }) }
-                  <div className="last-run-state title-3">
-                    { dashboard.last_run_state }
+                  <div className="last-run-state">
+                    <Flag
+                      caseType='sentenceCase'
+                      text={ dashboard.last_run_state }
+                      labelStyle={ dashboard.last_run_state === 'succeeded' ? 'success' : 'danger' }
```

Comment:
nit: Could move `succeeded` to constants file. Also could consider creating a utility function to map run-status to label style. We've only seen two statuses so far, but it's possible there are more statuses especially if we integrate other vendors. For example `succeeded, failed, waiting, canceled, in progress`.

CC: @jinhyukchang

Reply:
I will pull the `labelStyle` logic into its own method.

I'll file an issue for further discussion of some concerns around `last_run_state`, which IMO would be best to take action on only when we actually get to the point where we or a community member wants to integrate another vendor.